packages/plugin-seo/src/translations.ts

```diff
@@ -0,0 +1,23 @@
+export default () => {
```

Comment:
I'd like this to be made consistent with how we do translations in Payload.
We have a json file for each language and a schema file that enforces consistency. https://github.com/payloadcms/payload/blob/9da9b1fc5050d4f29bcf6dce2f22027834aaf698/packages/payload/src/translations/translation-schema.json

This way we can move towards having plugins contain all languages without developers needing to add their own translations project-by-project.

Reply:
Made the changes as discussed, added e2e tests for the plugin and for the new translations as well